modules/custom/social_graphql/tests/src/Kernel/MediaBridgeFileTest.php

```diff
@@ -59,8 +60,10 @@ protected function setUp() :void {
     // Give anonymous users permission to access content, so they can view and
     // download public files.
     $anonymous_role = Role::load(Role::ANONYMOUS_ID);
-    $anonymous_role->grantPermission('access content');
-    $anonymous_role->save();
+    if ($anonymous_role instanceof RoleInterface) {
+      $anonymous_role->grantPermission('access content');
+      $anonymous_role->save();
+    }
```

Comment:
Lets make this an assert, because not having the `anonymous` role would mean our Drupal instance is severely broken (or Drupal 11 has change in fun and interesting ways).